src/IdentityServer/src/IdentityServer.Core/Events/IdOpsEventSink.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Duende.IdentityServer.Events;
+using Duende.IdentityServer.Services;
+using IdOps.IdentityServer.Extensions;
+using Microsoft.Extensions.Logging;
+
+namespace IdOps.IdentityServer.Events
+{
+    public class IdOpsEventSink : IEventSink
+    {
+        private readonly ILogger<BusEventSink> _logger;
+        private readonly IEnumerable<IIdOpsEventSink> _sinks;
+
+        public IdOpsEventSink(ILogger<BusEventSink> logger, IEnumerable<IIdOpsEventSink> sinks)
+        {
+            _logger = logger;
+            _sinks = sinks;
+        }
+
+        public Task PersistAsync(Event evt)
+        {
+            try
+            {
+                Task.Run(() => ProcessAsync(evt)).Forget();
```

Comment:
Forget?

Reply:
It was like this in previous `BusEventSink`, seams that is fire and forget process